include/grpc++/impl/codegen/call.h

```diff
@@ -224,7 +263,7 @@ class CallOpSendMessage {
   /// after use.
   template <class M>
   Status SendMessage(const M& message,
-                     const WriteOptions& options) GRPC_MUST_USE_RESULT;
+                     WriteOptions options) GRPC_MUST_USE_RESULT;
```

Comment:
This was something I requested: we had places with WriteOptions&, and const WriteOptions&.
What we really needed everywhere was WriteOptions. And given this struct anyway needs to be tiny, let's just pass it as WriteOptions everywhere.